### PR TITLE
Figwheel portal example

### PR DESCRIPTION
**Pull request by olymk2**

### CODE REVIEW

**Comment by djblue** [comment]
Hi @olymk2, thanks for the PR! When I run the example project with `clj -Adev`, I don't seem to be getting a figwheel repl. Do you know what might be wrong?

File: examples/web-figwheel-main/readme.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Launch the demo with 
+
+    clj -Adev
```
Comment (djblue) [inline examples/web-figwheel-main/readme.md:3]: I think `clj -M:dev` is the new "correct" way to launch a clojure main function.

File: examples/web-figwheel-main/deps.edn [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{:paths ["src" "dev"]
```
Comment (djblue) [inline examples/web-figwheel-main/deps.edn:1]: This should probably be `["src" "resources"]` as `"dev"` is brought in via `:extra-paths` in the alias.

**Comment by olymk2** [comment]
> Hi @olymk2, thanks for the PR! When I run the example project with `clj -Adev`, I don't seem to be getting a figwheel repl. Do you know what might be wrong?

I think you need to put in --repl if you want the terminal repl else it just launches the browser

File: examples/web-figwheel-main/readme.md (hunk 1 above)
Comment (olymk2) [reply examples/web-figwheel-main/readme.md]: made this change it's hard to break habits :)

File: examples/web-figwheel-main/deps.edn (hunk 2 above)
Comment (olymk2) [reply examples/web-figwheel-main/deps.edn]: change made your correct

**Comment by djblue** [comment]
Hi @olymk2, adding `--repl` doesn't seem to work for me. It's like code isn't loading in the browser 🤔

**Comment by olymk2** [comment]
> Hi @olymk2, adding `--repl` doesn't seem to work for me. It's like code isn't loading in the browser thinking

try running clj -Adev I added in the repl options you need to use -A as -M only merges the main options and not the extra-deps keys. 

sorry it took a while

**Comment by djblue** [comment]
Ohh gotcha, that does seem to work for me! Thanks @olymk2!

**Pull request merged**